build/builtInExtensions.json

```diff
@@ -43,5 +43,20 @@
 			},
```

Comment:
This change will be removed. It was added because our extension depends on this extension to be installed.

Reply:
Removed.